float64[16] pred_grasps_cam
float64 score
float64[3] contact_pt